src/frontend/components/Comment.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component, PropTypes } from 'react';
+import Radium from 'radium';
+import {Paper, IconMenu, MenuItem, IconButton, FontIcon, ListItem, ListDivider, Avatar} from 'material-ui';
+import Colors from 'material-ui/lib/styles/colors'
+
+const styles = {
+
```

Comment:
Let's not leave any blank spaces at the top of blocks. I'll get a styleguide/linting in place for us soon. Most important is for us to agree on the rules and just be consistent. As an open source project, we'll need to be really tight on this stuff.